fly/commands/pipelines.go

```diff
@@ -78,17 +72,61 @@ func (command *PipelinesCommand) Execute([]string) error {
 			publicColumn.Contents = "no"
 		}
 
+		var archivedColumn ui.TableCell
+		if command.IncludeArchived {
+			if p.Archived {
+				archivedColumn.Contents = "yes"
+				archivedColumn.Color = ui.OnColor
+			} else {
+				archivedColumn.Contents = "no"
+			}
+		}
+
 		row := ui.TableRow{}
 		row = append(row, ui.TableCell{Contents: p.Name})
 		if command.All {
 			row = append(row, ui.TableCell{Contents: p.TeamName})
 		}
 		row = append(row, pausedColumn)
 		row = append(row, publicColumn)
+		if command.IncludeArchived {
+			row = append(row, archivedColumn)
+		}
 		row = append(row, ui.TableCell{Contents: time.Unix(p.LastUpdated, 0).String()})
 
 		table.Data = append(table.Data, row)
 	}
 
 	return table.Render(os.Stdout, Fly.PrintTableHeaders)
 }
+
+func (command *PipelinesCommand) buildHeader() []string {
+	var headers []string
+	if command.All {
+		headers = []string{"name", "team", "paused", "public"}
+	} else {
+		headers = []string{"name", "paused", "public"}
+	}
+
+	if command.IncludeArchived {
+		headers = append(headers, "archived", "last updated")
+	}
```

Comment:
I think we always want "last updated", right?

```suggestion
	if command.IncludeArchived {
		headers = append(headers, "archived")
	}
	headers = append(headers, "last updated")
```

Reply:
Wait, shouldn't the `fly/integration` tests be failing when `--include-archived` isn't provided, since it's no longer printing the `"last updated"` header?